internal/pkg/task/env_runner.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	fmtErrPublicSubnetsFromEnv  = "get public subnet IDs from environment %s: %w"
+	fmtErrPublicSubnetsFromEnv  = "get public subnet IDs from environment %s: %w "
```

Comment:
Can we revert this change? looks like an extra space was added at the end 🙏

Reply:
This is miss, sorry.